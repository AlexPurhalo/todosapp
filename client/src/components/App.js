import React from 'react'
import TodosForm from './TodosForm'
import TodosList from './TodosList'

export default function () { 
  const todos = [
    {id: 1, title: 'Call Mom',       isDone: false},
    {id: 2, title: 'Clean the room', isDone: true }
  ]
  const toggleTodoStatus = (id) => {
    console.log(id)
  }
  return (
    <div>
      <TodosForm />
      <TodosList todos={todos} toggleTodoStatus={toggleTodoStatus}/>
    </div>
  )
} 
