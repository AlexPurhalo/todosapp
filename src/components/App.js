import React, { Component } from 'react'
import { compose } from 'redux'
import { TodosFormContainer, TodosListContainer } from '../containers'
import withTodos from '../hocs/withTodos'

export default function (props) {
  return (
    <div>
      <TodosFormContainer />
      <TodosListContainer />
    </div>
  )
}
