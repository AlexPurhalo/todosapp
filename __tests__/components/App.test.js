import React from 'react'
import { shallow, mount  } from 'enzyme'
import App from '../../src/components/App'

describe('App container', () => {
  test('renders correct content', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
