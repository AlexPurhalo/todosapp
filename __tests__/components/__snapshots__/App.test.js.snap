// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App container renders correct content 1`] = `ShallowWrapper {}`;
